src/main/java/com/puppycrawl/tools/checkstyle/checks/design/MutableExceptionCheck.java

```diff
@@ -66,6 +66,130 @@
  * <pre>
  * &lt;module name=&quot;MutableException&quot;/&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class FirstClass extends Exception {
+ *   private int code; // OK, Class-Name doesn't match with default pattern
+ *
+ *   public FirstClass() {
+ *     code = 1;
+ *   }
+ * }
+ *
+ * class MyException extends Exception {
+ *   private int code; // violation, The field 'code' must be declared final
+ *
+ *   public MyException() {
+ *     code = 2;
+ *   }
+ * }
+ *
+ * class MyThrowable extends Throwable {
+ *    final int code;
+ *    String message; // violation, The field 'message' must be declared final
+ *
+ *    public MyThrowable(int code, String message) {
+ *      this.code = code;
+ *      this.message = message;
+ *    }
+ * }
+ *
+ * class ViolationException extends java.lang.Exception {
+ *   int code; // violation, The field 'code' must be declared final
+ *
+ *   public ViolationException(int code) {
+ *     this.code = code;
+ *   }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it only checks for Given Pattern in class-name:
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;MutableException&quot;&gt;
+ *   &lt;property name=&quot;format&quot; value=&quot;^.*Exception$&quot;/&gt;
+ * &lt;/module&gt;
+ * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * class FirstClass extends Exception {
+ *   private int code; // OK, Class-Name doesn't match with Given pattern
+ *
+ *   public FirstClass() {
+ *     code = 1;
+ *   }
+ * }
+ *
+ * class MyException extends Exception {
+ *   private int code; // violation, The field 'code' must be declared final
+ *
+ *   public MyException() {
+ *     code = 2;
+ *   }
+ * }
+ *
+ * class MyThrowable extends Throwable {
+ *   final int code; // OK
+ *   String message; // OK
+ *
+ *   public MyThrowable(int code, String message) {
+ *     this.code = code;
+ *     this.message = message;
+ *   }
+ * }
+ *
+ * class ViolationException extends java.lang.Exception {
+ *   int code; // violation, The field 'code' must be declared final
+ *
+ *   public ViolationException(int code) {
+ *     this.code = code;
+ *   }
+ * }
+ * </pre>
+ * <p>
+ * To configure the check so that it only checks for Given Pattern In SuperClassName:
```

Comment:
`... checks for type names used in 'extends' that match pattern`

Reply:
is any change require in this message : "OK, Extended-Class-Name doesn't match with Given pattern"

Reply:
@romani ?

Reply:
Done